File: app/src/main/java/com/mobidal/pharmacynamoune/model/Picture.java
package com.mobidal.pharmacynamoune.model;

import com.google.gson.annotations.SerializedName;

public class Picture {

    private int id;
    @SerializedName("picture_url")
    private String mPictureUrl;

    public int getId() {
        return id;
    }

    public String getPictureUrl() {
        return mPictureUrl;
    }
}

File: app/src/main/java/com/mobidal/pharmacynamoune/model/Picture.kt
package com.mobidal.pharmacynamoune.model;

import com.google.gson.annotations.SerializedName;

public class Picture {

    private int id;
    @SerializedName("picture_url")
    private String mPictureUrl;

    public Picture(int id, String mPictureUrl) {
        this.id = id;
        this.mPictureUrl = mPictureUrl;
    }

    public int getId() {
        return id;
    }

    public String getPictureUrl() {
        return mPictureUrl;
    }
}
